### ISSUE TITLE
Open in Explorer (feature)

### ISSUE THREAD

**Issue by Zexx1000** [geany/geany-plugins#380]
One much needed feature in Geany is to right-click on a tab (of the currently edited file) and choose "Open File Location". 

If it doesn't exist in Geany, it should exist in TreeBrowser. Right-click on a file and then "Open File Location" should open the File Manager in the file's containing folder. This is especially important on Linux, since neither FileOpen nor FileSave dialogs on Linux have the advanced functionality that is normal on Windows.

TreeBrowser adds many of those functions (such as renaming or deleting) but sometimes I just need to open File Manager in that folder and do other tasks. I think majority of users would profit from this option.

**Comment by codebrainz** [comment]
This seems like it would be a reasonable feature to implement in the core (and kind of hard to implement well from a plugin). I could imagine adding a "Tool" to "Preferences->Tools->Tool Paths" for the file manager (ex. `start %d` in Windows, `open %d` in OSX, and something like `exo-open %d` in others, where `%d` is a directory path), and then adding items to the document tab and document sidebar context menu for "Open in File Manager..." that use that tool command. Maybe also adding a keybinding, and perhaps an API function for plugins like TreeBrowser too support the same functionality.

**Comment by Zexx1000** [comment]
I agree it should be in the core. In fact this entire plugin should be in the core. But since it's not, and since the plugin is kinda "file manager" itself, with some useful OS functions, adding a command to open a default file manager would be quite aligned with the rest of its functionality.

**Comment by frlan** [comment]
Treebrowser is having this already 
![geany_treebrowser_openextern](https://cloud.githubusercontent.com/assets/1010248/13532356/631e2672-e22b-11e5-8d85-b33be3e02a46.png)

**Comment by frlan** [comment]
I tent to close this issue as it's already there. Maybe rewording would be a good thing -- not sure. Please reopen if you like
